cmd/minikube/cmd/config/config.go

```diff
@@ -91,6 +91,14 @@ var settings = []Setting{
 		name: "kubernetes-version",
 		set:  SetString,
 	},
+	{
+		name: "dashboard",
```

Comment:
I don't understand what this should look like, could you perhaps give an example?  Thanks.